precompiles/ArbAggregator.go

```diff
@@ -20,87 +16,111 @@ type ArbAggregator struct {
 	Address addr // 0x6d
 }
 
-// Gets an account's preferred aggregator
+var ErrNotOwner = errors.New("must be called by chain owner")
+
+// [Deprecated]
 func (con ArbAggregator) GetPreferredAggregator(c ctx, evm mech, address addr) (prefAgg addr, isDefault bool, err error) {
-	l1p := c.State.L1PricingState()
-	maybePrefAgg, err := l1p.UserSpecifiedAggregator(address)
+	posters, err := c.State.L1PricingState().BatchPosterTable().AllPosters()
 	if err != nil {
 		return common.Address{}, false, err
 	}
-	if maybePrefAgg != nil {
-		return *maybePrefAgg, false, nil
-	}
-	maybeReimbursableAgg, err := l1p.ReimbursableAggregatorForSender(address)
-	if err != nil || maybeReimbursableAgg == nil {
-		return common.Address{}, false, err
+	if len(posters) == 0 {
+		return common.Address{}, false, errors.New("no batch posters exist")
 	}
-	return *maybeReimbursableAgg, true, nil
+	return posters[0], true, err
```

Comment:
How about we return `l1pricing.BatchPosterAddress` here? I think that's a bit more sensible than returning effectively a random aggregator.